### PR TITLE
Create primary_sticker_image_url in SiteConfig

### PR DESCRIPTION
**Pull request by JibranKalia**
## What type of PR is this? (check all applicable)

- [ ] Refactor
- [X] Feature
- [ ] Bug Fix
- [ ] Optimization
- [ ] Documentation Update

## Description
Taking a first pass at adding logo images to Site Config. Wanted to just start with `primary_sticker_image_url` to make sure my approach is correct. Will complete the rest in a different PR if this one is acceptable. 

## Related Tickets & Documents
https://github.com/thepracticaldev/dev.to/issues/5477

## Mobile & Desktop Screenshots/Recordings (if there are UI changes)
Just chose a random svg to change the logo: 

Screen Shot 2020-03-20 at 9 54 07 AM

Screen Shot 2020-03-20 at 9 54 23 AM

## Added tests?

- [X] yes
- [ ] no, because they aren't needed
- [ ] no, because I need help

## Added to documentation?

- [ ] docs.dev.to
- [ ] readme
- [ ] no documentation needed

### CODE REVIEW

File: app/models/site_config.rb [hunk 1]
```diff
@@ -29,6 +29,7 @@ class SiteConfig < RailsSettings::Base
   field :main_social_image, type: :string, default: "https://thepracticaldev.s3.amazonaws.com/i/6hqmcjaxbgbon8ydw93z.png"
   field :favicon_url, type: :string, default: "favicon.ico"
   field :logo_svg, type: :string, default: ""
+  field :primary_sticker_image_url, type: :string, default: "https://practicaldev-herokuapp-com.freetls.fastly.net/assets/rainbowdev.svg"
```
Comment (JibranKalia) [inline app/models/site_config.rb]: Just want to make sure this is the url we want to be default.

Comment (Zhao-Andy) [reply app/models/site_config.rb]: I think it's fine; thoughts @benhalpern?

**Comment by benhalpern** [comment]
I'm going to leave this for our internal generalization-focused team to give a final pass on this Monday, but I feel like it definitely works for me.

One thing that doesn't have to hold this PR up because the underlying stuff works and this change could come with a quick PR after, but when rendered in the view, this should probably be wrapped in `cloudinary(url, width: xxx)` because if a community admin adds a URL we probably shouldn't render that raw URL in case it is a slow CDN or way too big or whatever. Wrapping in `cloudinary` will ensure it always is the right rendering experience for the end user.

**Comment by JibranKalia** [comment]
Thank You both for the review. Great point about cloudinary. I will go ahead and add it on the next PR I am working on.

**Comment by JibranKalia** [comment]
Hi @rhymes. Thank You. Had fun writing it.

So I considered that as well when writing the PR. Here are my thoughts:

Anything that expects a url for example `main_social_image_url` `primary_sticker_image_url` should have the `_url` suffix. On the other hand where an svg needs to be pasted in it should be just `logo_svg`. And then appropriate validations can be added to make sure the correct input is entered.

Just my two cents. But I am open to whatever you decide and can update my PR accordingly.

**Comment by benhalpern** [comment]
I merged this because I think we're going to have to reconcile several different names in order to truly get to a convention we're all happy with, and once we do we can migrate everything at once.

Getting this one just right wasn't going to unify everything, so once we have more decisions under our belt we can look at a unification PR for conventionality.

**Comment by rhymes** [comment]
@JibranKalia your reasoning makes a lot of sense to me! I agree, the names should be explicit. It's harder to know if one should provide or upload an image instead of filling in a URL otherwise. Thanks for your thoughts!

@benhalpern agreed :)

**Pull request merged**